examples/mongodb/locustfile.py

```diff
@@ -0,0 +1,13 @@
+from locust import between, task
+from locust.contrib.mongodb import MongoDBUser
+
+import os
+
+
+class MongoDBUser(MongoDBUser):
+    conn_string = os.getenv("MONGODB_URI")
```

Comment:
maybe leave this hardcoded? reading from env isn't really key to this example :)

Reply:
Yeah good call😊